frontend/src/pages/modelServing/screens/projects/EmptyMultiModelServingCard.tsx

```diff
@@ -49,7 +49,11 @@ const EmptyMultiModelServingCard: React.FC = () => {
   return (
     <>
       <Card
-        style={{ height: '100%', border: '1.5px dashed var(--pf-v5-global--BorderColor--200)' }}
+        style={{
+          height: '100%',
+          border: '1px solid var(--pf-v5-global--BorderColor--100)',
+          borderRadius: 16,
```

Comment:
When I check the border radius for cards on the Overview tab, they're set to `10` instead of `16`

Reply:
Please check again @jgiardino. I'm not sure the cluster was updated when you checked. Looking now (uxdpoc3, project Jenn 2) I see a 16px radius.

Reply:
Fixed here: https://github.com/opendatahub-io/odh-dashboard/compare/e7aab0fc8a818914c439126d4aa867fa711eaf66..f1206e75a75a781eb045a377969c56692eaf8723#diff-27e6c1b788771fb76b7d4d17328f2b51e3eb8a330b59135a3c4e8ab84b4599d9R3